### DISCUSSION TITLE
Expected output when all POS are applicable?

### DISCUSSION THREAD

**Discussion by brbucher** [CMSgov/price-transparency-guide#386] [Q&A]
What is the expected output when all place of service codes are applicable?
There are a few mentions of this in the overall POS discussion here: [https://github.com/CMSgov/price-transparency-guide/discussions/63](https://github.com/CMSgov/price-transparency-guide/discussions/63)
Would the expectation be that each POS type is listed out individually or would an empty array or "All" denotation be possible?

Allowing for empty arrays or "All" could save a significant amount of space in the files.

**Comment by brbucher** [comment]
A value like "[CSTM-00](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#additional-notes-concerning-billing_code)" would be very useful here.  Could we have something similar for service codes?

**Comment by brbucher** [comment]
Bumping this thread, any thoughts @shaselton?

**Comment by rakeshkatte** [comment]
Service_code is not a required node, so when the rate applies to all POS or
in case where the rate is not dependent on POS then we are ignoring this
node.

**Reply by brbucher** [reply]
[https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object)
Service code is required when the billing class is professional, but we are following the same approach for institutional.

**Comment by rakeshkatte** [comment]
My bad, you are right. For professional we are listing down all POS in the
array.
